content/en/docs/tasks/cluster-administration/node-overprovisioning/node-overprovisioning.md

```diff
@@ -35,4 +33,104 @@ metadata:
 value: 1000
 globalDefault: false
 description: "Low priority for placeholder pods to enable overprovisioning."
-```
\ No newline at end of file
+```
+
+Then apply the Priority Class:
+
+```shell
+kubectl apply -f low-priority-class.yaml
+```
+
+Now, to create the Placeholder Deployment, define a deployment that uses the low-priority Priority Class and runs a minimal container.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: placeholder-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: placeholder
+  template:
+    metadata:
+      labels:
+        app: placeholder
+    spec:
+      priorityClassName: low-priority
+      containers:
+      - name: pause
+        image: k8s.gcr.io/pause:3.2
+        resources:
+          requests:
+            cpu: "50m"
+            memory: "100Mi"
+          limits:
+            cpu: "50m"
+            memory: "100Mi"
+```
+
+Apply the deployment:
+
+```shell
+kubectl apply -f placeholder-deployment.yaml
+```
+
+## Adjust Resource Requests and Limits
+
+Configure the resource requests and limits for the placeholder pods to define the amount of overprovisioned resources you want to maintain. This reservation ensures that a specific amount of CPU and memory is kept available for new pods.
+
+To edit the Deployment Configuration, modify the resources section in the Deployment YAML to set appropriate requests and limits. For example, to reserve 500m CPU and 1Gi memory across all placeholder pods:
+
+```yaml
+resources:
+  requests:
+    cpu: "100m"
+    memory: "200Mi"
+  limits:
+    cpu: "100m"
+    memory: "200Mi"
+```
+
+Now, to adjust the replicas count accordingly to achieve the desired total reserved resources, update the Deployment with the new resource configurations ensuring the aggregate of all placeholder pods meets your overprovisioning targets:
```

Comment:
There is no adjustment to replicas, am I missing something?

Reply:
I will remove this